specifyweb/frontend/js_src/lib/components/ExportFeed/Editor.tsx

```diff
@@ -330,11 +330,11 @@ function ResourcePicker({
             filters={dwcaAppResourceFilter}
             header={title}
             resources={resources}
-            skippable={type === 'metadata'}
```

Comment:
if `skippable` prop is no longer used here, can you check if it is still used anywhere, or else if it can be removed?

Reply:
The `skippable` prop is still used for the `Create DwC Archive` tool. 
https://github.com/specify/specify7/blob/bb328a30a92eae17a13854356c481a5163e14759/specifyweb/frontend/js_src/lib/components/ExportFeed/Dwca.tsx#L50-L60.

Specifically, a DwC Archive can be created without an accompanying Metadata file. 

https://github.com/specify/specify7/assets/64045831/38b38c08-5825-4468-83d8-493ec3d719b0